src/App.js

```diff
@@ -1,15 +1,28 @@
 import React from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
+import Restaurant from "./Restaurant";
+import TouristInfoCards from "./TouristInfoCards";
+import FooterCompo from "./FooterCompo";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
-      <Bookings />
+      <Heading />
+      <TouristInfoCards />
+      <Bookings name />
+      <Restaurant />
+      <FooterCompo
+        details={[
```

Comment:
For different reasons(please research), I would suggest make App.js file as light as possible. 
What's the point of passing `details` as props to the component?
So how about if you move `details` to `FooterCompo` and then `map` on the array of details?